front_end/panels/ai_chat/ui/AIChatPanel.ts

```
@@ -2205,22 +2182,105 @@ export class AIChatPanel extends UI.Panel.Panel {
+      this.performUpdate();
+
+      logger.info('Conversation loaded from history', {conversationId});
+    } else {
+      logger.error('Failed to load conversation', {conversationId});
+    }
+  }
+
+  /**
+   * Starts a new conversation
+   */
+  async #startNewConversation(): Promise<void> {
```

ESLint

[error] '#startNewConversation' is defined but never used.

(no-unused-private-class-members)